package.json

```diff
@@ -12,26 +12,26 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^16.2.10",
+    "@angular/animations": "^18.2.7",
     "@angular/cdk": "^16.2.9",
-    "@angular/common": "^16.2.10",
-    "@angular/compiler": "^16.2.10",
-    "@angular/core": "^16.2.10",
-    "@angular/forms": "^16.2.10",
+    "@angular/common": "^18.2.7",
+    "@angular/compiler": "^18.2.7",
+    "@angular/core": "^18.2.7",
+    "@angular/forms": "^18.2.7",
     "@angular/material": "^16.2.9",
```

Comment:
Do not forget to update angular material as well...

Reply:
hello @whyvrafvr  done .